docs/book/src/SUMMARY.md

```diff
@@ -118,6 +118,7 @@
   - [Extending the CLI](./plugins/extending-cli.md)
   - [Creating your own plugins](./plugins/creating-plugins.md)
   - [Plugins Versioning](./plugins/plugins-versioning.md)
+  - [Write Tests For Your Own Plugin](./plugins/test-plugin.md)
```

Comment:
What about it be under `Creating your own plugins`

Reply:
Yeah, sounds good. 👍🏼  I misunderstood the origin idea previously...